src/test/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/UserControllerIntTest.kt

```diff
@@ -791,4 +791,51 @@ class UserControllerIntTest : IntegrationTestBase() {
         .expectStatus().isUnauthorized
     }
   }
+
+  @Nested
+  inner class MyCaseloads {
+
+    private val userId: UUID = UUID.fromString("5E3850B9-9D6E-49D7-B8E7-42874D6CEEA8")
+
+    @Test
+    fun `get caseloads when no authority`() {
+      webTestClient.get().uri("/users/me/caseloads")
+        .exchange()
+        .expectStatus().isUnauthorized
+    }
+
+    @Test
+    fun `get user caseloads`() {
+      val username = "AUTH_ADM"
```

Comment:
Tiny comment are we able to use a different user for this test AUTH_ADM is an external user so could get confusing at a later date